NAMESPACE

```diff
@@ -9,6 +9,7 @@ S3method(create_sparklines,factor)
 S3method(create_sparklines,logical)
 S3method(create_sparklines,numeric)
 export(add_facet_labels)
+export(crule)
```

Comment:
does it have to be exported?

Reply:
this will go into `teal` so it will be exported there to be used in all repos.
@chlebowa is working on this.

Reply:
This function is being moved to `teal`. It will be exported from there.

Reply:
This is outdated, `crule` will not be added, all changes are being rolled back.